### ISSUE TITLE
test_chain_selection return python error

### ISSUE THREAD

**Issue by tubiana** [BradyAJohnston/MolecularNodes#27]
Hi again :) 

I wanted to try the "test_chain_selection" node but it return a python error when an MD is loaded (I haven't tried yet with a static PDB)

```python
Python: Traceback (most recent call last):
  File "C:\Users\tubia\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\molecular_nodes\__init__.py", line 604, in invoke
    return self.execute(context)
  File "C:\Users\tubia\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\molecular_nodes\__init__.py", line 417, in execute
    bpy.ops.wm.append(
  File "C:\Program Files\Blender Foundation\Blender 3.1\3.1\scripts\modules\bpy\ops.py", line 132, in __call__
    ret = _op_call(self.idname_py(), None, kw)
RuntimeError: Error: 'C:\Users\BradyJohnston\Documents\GitHub\MolecularNodes\molecular_nodes_append_file.blend\NodeTree\MOL_utils_bool_chain': not a library


location: <unknown location>:-1
```
Cheers!
Thibault.

**Comment by BradyAJohnston** [comment]
Thanks for reporting!

**Comment by BradyAJohnston** [comment]
Fixed in [v0.5.21](https://github.com/BradyAJohnston/MolecularNodes/releases/tag/v0.5.21)
